pages/index.js

```diff
@@ -116,6 +117,31 @@ const GraphNumber = s(Title)`
   font-size: 50px;
 `
 
+const GraphNumberBubble = s.div`
+  line-height: 1;
+  transform: translate(0, 20%);
+  color: #D12D4A;
+`
+
+const StyledAnchor = s.a`
+  color: #000000 !important;
+  text-decoration: none !important;
+`
+const NewsLetterWrapper = s.div`
+  .newsletter {
+    justify-content: center;
+    margin-top: 4rem;
+    padding: 0 15rem;
+    @media(max-width: 768px) {
+      padding: 0 2rem;
+    }
```

Comment:
why don't you just style the row/elements with .newsletter using styled-components directly? I could understand if you were styling a component from a library, but afaik .newsletter is being applied only to vanilla diffs.